src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -46,108 +54,155 @@
 
+        // Lists
+          List<User> users = userStore.getLatestUsers();
+          List<Conversation> conversations = (List<Conversation>) convoStore.getLatestConversations();
+          List<Message> messages = (List<Message>) request.getAttribute("messages");
 
+        // Master List
 
+          /*
+            Overall Format:
+              Date | Mode | Username | Convo | Message Content
 
+            Convo Format:
+              Date | c | Username | Convo | NULL
 
+            User Format:
+              Date | u | Username |  NULL | NULL
 
+            Message Format:
+              Date | m | Username | Convo | Message Content
+          */
 
+          int totalSize = conversations.size() + users.size() + messages.size();
+          String[][] masterList = new String[totalSize][5];
 
+          // add users
+          for (int i = 0; i < users.size(); i++) {
 
+              User currUser = users.get(i);
 
-      <!-- CONVERSATION CREATION -->
-      <h3>New Conversations</h3>
-     
-      <%
-      // Pull conversation data
-      List<Conversation> conversations = (List<Conversation>) request.getAttribute("conversations");
-      ListIterator<Conversation> itrC = conversations.listIterator(conversations.size());
-      UserStore allUsers = (UserStore) request.getAttribute("users");
+              masterList[i][0] = currUser.getCreationTime().toString();
+              masterList[i][1] = "u";
+              masterList[i][2] = currUser.getName();
+              masterList[i][3] = "";
+              masterList[i][4] = "";
+          }
 
-      // empty?
-      if(conversations == null || conversations.isEmpty()){
-      %>
-        <p>No New Conversations.</p>
-      <%
-      }
-      else{ // not empty? make a list
-      %>
-        <ul class="mdl-list">
-      <%
-        while(itrC.hasPrevious()){
-          Conversation currConvo = itrC.previous();
-      %>
-        <li>
-          <strong> <%= currConvo.getCreationTime() %>: </strong> 
-          <%= (allUsers.getUser(currConvo.getOwnerId())).getName() %> created a new conversation: 
-          <a href="/chat/<%= currConvo.getTitle() %>"> <%= currConvo.getTitle() %></a>
-        </li> 
-      <%
-        }
-      %>
-        </ul>
-      <%
-      }
-      %>
+          // add convos
+          for (int i = 0; i < conversations.size(); i++) {
 
-      <!-- NEW MESSAGES -->
-      <h3>New Messages</h3>
-     
+              Conversation currConvo = conversations.get(i);
+              int buff = users.size();
 
-    </div>
+              masterList[i+buff][0] = currConvo.getCreationTime().toString();
+              masterList[i+buff][1] = "c";
+              masterList[i+buff][2] = (userStore.getUser(currConvo.getOwnerId())).getName();
+              masterList[i+buff][3] = currConvo.getTitle();
+              masterList[i+buff][4] = "";
+          }
+
+          // // add messages
+          for (int i = 0; i < messages.size(); i++) {
+
```

Comment:
nitpick: same as above — i _feel_ like usually there's no newline between the first statement of the for loop and the initializing line (but someone correct me if i'm wrong!)